src/txmempool.h

```diff
@@ -476,7 +476,7 @@ enum class MemPoolRemovalReason {
  */
 class CTxMemPool
 {
-private:
+protected:
```

Comment:
It seems a bit invasive to change all of these mempool members... can you just call something that automatically does `RollingFeeUpdate` like `tx_pool.GetMinFee()`? Or just set the members you need to `protected`?

Reply:
`tx_pool.GetMinFee` exists early because of `!blockSinceLastRollingFeeBump`, so I can't use that. `protected` as a replacement for `private` seems fine generally, because it allows tests to mock any private member without having to mark each member individually or add a `friend TestClass01` for each class that mocks the mempool. Since mempool isn't derived outside of tests, `protected` shouldn't come with any risks either.

Reply:
Sure, makes sense 👍